src/main.cc

```diff
@@ -86,6 +87,28 @@ class Spellchecker : public Nan::ObjectWrap {
     }
   }
 
+  static NAN_METHOD(CheckSpellingCallback) {
+    Nan::HandleScope scope;
+    if (info.Length() < 2) {
+      return Nan::ThrowError("Bad argument");
+    }
+
+    Handle<String> string = Handle<String>::Cast(info[0]);
+    if (!string->IsString()) {
+      return Nan::ThrowError("Bad argument");
+    }
+
+    Nan::Callback *callback = new Nan::Callback(info[1].As<v8::Function>());
+
+    std::vector<uint16_t> *corpus = new std::vector<uint16_t>(string->Length() + 1);
```

Comment:
Oh man, good catch. Also, this is the first time I've seen `std::move`. Hooray, I learned a thing :tada: